security-command-center/snippets/src/main/java/management/api/CreateSecurityHealthAnalyticsCustomModule.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package management.api;
+
+// [START securitycenter_management_api_create_security_health_analytics_custom_module]
+
+import com.google.cloud.securitycentermanagement.v1.CreateSecurityHealthAnalyticsCustomModuleRequest;
+import com.google.cloud.securitycentermanagement.v1.CustomConfig;
+import com.google.cloud.securitycentermanagement.v1.CustomConfig.ResourceSelector;
+import com.google.cloud.securitycentermanagement.v1.CustomConfig.Severity;
+import com.google.cloud.securitycentermanagement.v1.SecurityCenterManagementClient;
+import com.google.cloud.securitycentermanagement.v1.SecurityHealthAnalyticsCustomModule;
+import com.google.cloud.securitycentermanagement.v1.SecurityHealthAnalyticsCustomModule.EnablementState;
+import com.google.type.Expr;
+import java.io.IOException;
+
+public class CreateSecurityHealthAnalyticsCustomModule {
+
+  public static void main(String[] args) throws IOException {
+
+    // parent: Use any one of the following options:
+    // - organizations/{organization_id}/locations/{location_id}
+    // - folders/{folder_id}/locations/{location_id}
+    // - projects/{project_id}/locations/{location_id}
+    String parent = String.format("organizations/%s/locations/%s", "organization_id", "global");
+
+    // display name of the module, update with your name
+    String customModuleDisplayName = "custom_module_display_name";
```

Comment:
no need for this input argument to run the code sample. move the display name into the method.

Reply:
Actually I added this input arguments because in all test cases I am passing the custom modules name explicitly. Before running any test case I am creating one custom module and performing the test accordingly.